Model/PaymentMethod/AlipayCrossBorderPaymentMethod.php

```diff
@@ -74,6 +74,7 @@ public function getTransaction()
      */
     public function addMandatoryTransactionData(&$oTransaction, $oOrder)
     {
+        //TODO: once account holder is not needed by the sdk, remove this line
```

Comment:
What is meant by this?

Reply:
The TODO is a reminder to remove the account holder in the future once it is not needed anymore. This was discussed yesterday.